@Inject with @Named causing the AWS Lambda bootstrap failure

**Describe the bug**
I am getting Lambda Bootstrap Error when using **@Inject and @Named** on RequestHandler

**Expected behavior**
The CDI injection & Lambda functionality should work as described in the documentation.

**Actual behavior**
Bootstrap error and Lambda function doesn't work

```text

{
  "errorMessage": "Quarkus bootstrap failed.\njava.lang.RuntimeException: Failed to start quarkus\n\tat io.quarkus.runner.ApplicationImpl.doStart(ApplicationImpl.zig:208)\n\tat io.quarkus.runtime.Application.start(Application.java:89)\n\tat io.quarkus.amazon.lambda.runtime.QuarkusStreamHandler.<clinit>(QuarkusStreamHandler.java:39)\n\tat java.base/java.lang.Class.forName0(Native Method)\n\tat java.base/java.lang.Class.forName(Unknown Source)\n\tat lambdainternal.HandlerInfo.fromString(HandlerInfo.java:31)\n\tat lambdainternal.AWSLambda.findUserMethods(AWSLambda.java:77)\n\tat lambdainternal.AWSLambda.startRuntime(AWSLambda.java:196)\n\tat lambdainternal.AWSLambda.startRuntime(AWSLambda.java:162)\n\tat lambdainternal.AWSLambda.main(AWSLambda.java:157)\nCaused by: java.lang.RuntimeException: Unable to find handler class with name test make sure there is a handler class in the deployment with the correct @Named annotation\n\tat io.quarkus.amazon.lambda.runtime.AmazonLambdaRecorder.chooseHandlerClass(AmazonLambdaRecorder.java:105)\n\tat io.quarkus.deployment.steps.AmazonLambdaProcessor$recordHandlerClass16.deploy_0(AmazonLambdaProcessor$recordHandlerClass16.zig:210)\n\tat io.quarkus.deployment.steps.AmazonLambdaProcessor$recordHandlerClass16.deploy(AmazonLambdaProcessor$recordHandlerClass16.zig:229)\n\tat io.quarkus.runner.ApplicationImpl.doStart(ApplicationImpl.zig:139)\n\t... 9 more\n",
  "errorType": "java.io.IOException",
  "stackTrace": [
    "io.quarkus.amazon.lambda.runtime.QuarkusStreamHandler.handleRequest(QuarkusStreamHandler.java:56)",
    "java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)",
    "java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)",
    "java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)",
    "java.base/java.lang.reflect.Method.invoke(Unknown Source)"
  ]
}

```

**To Reproduce**
Steps to reproduce the behavior:
1.  Create a AWS lambda project using Quakus maven architype
2. Create two classes like below

```java
@ApplicationScoped
@Slf4j
public class CDIConfig {

    private final String prop1;

    @Inject
    public CDIConfig(
            @ConfigProperty(name = "test.prop1") final String prop1) {

        this.prop1 = prop1;
    }

    @Produces
    @Singleton
    @Named("prop1")
    public String prop1() {
        log.info("Bean Creation:{}", prop1);
        return prop1;
    }

}
```

```java

@Named("test")
@Slf4j
public class TestLambda implements RequestHandler<InputObject, OutputObject> {

    @Inject
    ProcessingService service;

    @Inject
    @Named("prop1")
    String prop1;

    @Override
    public OutputObject handleRequest(final InputObject input, final Context context) {
        log.info("Prop1 in lambda:{}", prop1);
        return service.process(input).setRequestId(context.getAwsRequestId());
    }
}

```
3. mvn clean package and deploy the function to aws and run the test to see the error

**Configuration**
```properties
quarkus.lambda.handler=test
test.prop1=${GREETINGS:Hello}
```

**Screenshots**
(If applicable, add screenshots to help explain your problem.)

**Environment (please complete the following information):**
 - Output of `uname -a` or `ver`: macos
 - Output of `java -version`:  openjdk version "11.0.6" 2020-01-14
 - GraalVM version (if different from Java):  OpenJDK 64-Bit Server VM GraalVM CE 19.3.1 (build 11.0.6+9-jvmci-19.3-b07, mixed mode, sharing)
 - Quarkus version or git rev: OpenJDK 64-Bit Server VM GraalVM CE 19.3.1 (build 11.0.6+9-jvmci-19.3-b07, mixed mode, sharing)
 - Build tool (ie. output of `mvnw --version` or `gradlew --version`): mvn

**Additional context**
(Add any other context about the problem here.)

--- a/extensions/amazon-lambda/deployment/src/main/java/io/quarkus/amazon/lambda/deployment/AmazonLambdaProcessor.java
+++ b/extensions/amazon-lambda/deployment/src/main/java/io/quarkus/amazon/lambda/deployment/AmazonLambdaProcessor.java
@@ -102,9 +102,9 @@ public final class AmazonLambdaProcessor {
             reflectiveClassBuildItemBuildProducer.produce(new ReflectiveClassBuildItem(true, false, lambda));
 
             String cdiName = null;
-            List<AnnotationInstance> named = info.annotations().get(NAMED);
-            if (named != null && !named.isEmpty()) {
-                cdiName = named.get(0).value().asString();
+            AnnotationInstance named = info.classAnnotation(NAMED);
+            if (named != null) {
+                cdiName = named.value().asString();
             }
 
             ClassInfo current = info;

